src/features/walletconnect/components/WcSessionMananger/index.tsx

```diff
@@ -25,11 +25,14 @@ const WcSessionManager = ({ sessions, uri }: WcSessionManagerProps) => {
   const { walletConnect, error, setError, open, setOpen } = useContext(WalletConnectContext)
   const [proposal, setProposal] = useState<Web3WalletTypes.SessionProposal>()
   const [changedSession, setChangedSession] = useState<SessionTypes.Struct>()
+  const isResponded = useRef<boolean>(false)
```

Comment:
Do we need to reset this back to false when the proposal changs? Or is the entire component re-rendering then?

Reply:
Good point! Fixed.

Reply:
Thanks!

Btw please don't resolve threads unless you started them so that the author can come back and see your response (and hopefully resolve it) 😄